### PR TITLE
get dev images optimized from cloudinary

### PR DESCRIPTION
**Pull request by 29avet1**
## Description of Problem
jpg images load longer
## Proposed Solution
get size-optimized images from cloudinary storage in webp format

### CODE REVIEW

**Review by dionvogliqi — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .DS_Store
 .AppleDouble
 .LSOverride
+.idea
```
Comment (borna9II) [inline .gitignore:6]: remove this line

File: src/developers/components/CloudinaryImage.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  faceRecognition?: {
+    type: boolean,
+    default: false,
+  },
+  loading?: 'lazy' | 'eager',
+  className?: {
+    type: string,
+    default: '',
+  },
```
Comment (borna9II) [inline src/developers/components/CloudinaryImage.vue:25]: remove commas

File: src/developers/components/CloudinaryImage.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}>()
+
+const cloudinaryUrl = 'https://res.cloudinary.com/proxify-io/image/upload'
```
Comment (borna9II) [inline src/developers/components/CloudinaryImage.vue:28]: move to config file

File: src/developers/components/CloudinaryImage.vue [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const props = defineProps<{
```
Comment (TamaraZoric) [inline src/developers/components/CloudinaryImage.vue:4]: @borna9II @29avet1 This is almost correct vue 3 syntax. If you want to add default values for the props we should use

```
const props = withDefaults(defineProps<{
  src: string
  alt: string
  width: number
  height: number
  quality?: string
  crop?: string
  faceRecognition?: boolean
  loading?: 'lazy' | 'eager'
  class?: string
  ....
}>(), {
  quality: 'q_auto:best',
  crop: 'c_fit',
  faceRecognition: false,
  class: '',
  ....
});

```

I do not know it if we want to update this and also if PR to Vue is published.

**Pull request merged**